src/pages/PromptPage/PromptPage.jsx

```diff
@@ -8,12 +8,13 @@ import Roadmap from "../../components/Roadmap/Roadmap";
 import MatchedUsers from "../../components/MatchedUsers/MatchedUsers";
 import LoadingPage from "../../components/LoadingPage/LoadingPage";
 import NoMatch from "../../components/NoMatch/NoMatch";
+import GoalAchieved from "../../components/GoalAchieved/GoalAchieved";
 
 const PromptPage = () => {
-  const [currentPage, setCurrentPage] = useState("prompt");
+  const [currentPage, setCurrentPage] = useState("goalachieved");
 
   useEffect(() => {
-    setCurrentPage("prompt");
+    setCurrentPage("goalachieved");
```

Comment:
Change this to prompt

Reply:
It looks great, Jagadeep! You can change it back to prompt for current page.